### ISSUE TITLE
after conda update of jupyter - matplotlib DLL load failed 

### ISSUE THREAD

**Issue by dwervin** [conda/conda#9493]
Tom Caswell on matplotlib suggested this is an issue with conda as it uses precompiled binaries.

Any ideas?

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-2-ffe729860a8a> in <module>
      2 #%matplotlib inline
      3 
----> 4 import matplotlib.pyplot as plt
      5 import numpy as np
      6 import pandas as pd

C:\ProgramData\Anaconda3\envs\py37\lib\site-packages\matplotlib\__init__.py in <module>
    203 
    204 
--> 205 _check_versions()
    206 
    207 

C:\ProgramData\Anaconda3\envs\py37\lib\site-packages\matplotlib\__init__.py in _check_versions()
    188     # Quickfix to ensure Microsoft Visual C++ redistributable
    189     # DLLs are loaded before importing kiwisolver
--> 190     from . import ft2font
    191 
    192     for modname, minver in [

ImportError: DLL load failed: The specified module could not be found.
```

**Comment by msarahan** [comment]
From the issue template:


This is an issue tracker for conda -- the package manager.  File feature requests
for conda here, as well as bug reports about something conda has messed up.

If your issue is a bug report for:
  * a specific conda package from Anaconda ('defaults' channel):
      ==> file at https://github.com/ContinuumIO/anaconda-issues
  * a specific conda package from conda-forge:
      ==> file at the corresponding feedstock under https://github.com/conda-forge
  * repo.anaconda.com access and service:
      ==> file at https://github.com/ContinuumIO/anaconda-issues
  * anaconda.org access and service:
      ==> file at https://anaconda.org/contact/report
  * commands under 'conda build':
      ==> file at https://github.com/conda/conda-build
  * commands under 'conda env':
      ==> please file it here!
  * all other conda commands that start with 'conda':
      ==> please file it here!

**Comment by dwervin** [comment]
This was for conda.  According to Tom Caswell on matplotlib, he indicated that conda must have installed an invalid precompiled binary.  Since nothing else changed but me running conda to update Jupyter, seemed like a valid failure.

**Comment by dwervin** [comment]
Seems that Tom's concern was correct.  I think conda is not managing dependencies correctly.

It appears that I have the issue resolved.  Let me know if you want to know how the issue was resolved.  Thanks

**Comment by msarahan** [comment]
Sure, please tell us more about what you found. If nothing else, your
information here may help someone later who encounters the same problem.

It's rare that conda manages dependencies incorrectly, but fairly common
that package metadata is incorrect and leads conda to bad choices. It's
also reasonably common that packaging mistakes put incorrect files into
packages or put correct files into the wrong place. These are not conda
problems. These are problems with the packages that conda consumes, and
they need to be addressed on the package recipes. That's why the issue
template asks you to carefully consider where your issue belongs.

**Comment by dwervin** [comment]
Is there some reason why running "conda update jupyter" screwed up matplotlib?  Because that's all I did.

**Comment by msarahan** [comment]
You have given us nowhere near enough information to say anything.

**Comment by dwervin** [comment]
I told you exactly what happened.  It was working fine.  I ran conda update jupyter.  After that import matplotlib failed.  That's all there is to tell.  What else can I provide?

**Comment by dwervin** [comment]
Original install of Python and matplotlib was all done by Anaconda install.  I really don't know what else I can say.

**Comment by msarahan** [comment]
Again, from the issue template:

If you continue on,

   **please include all requested information below.**

If a maintainer determines the information is required to understand 
your issue, and if it is not provided, your issue may be
closed automatically.

-->

## Current Behavior



### Steps to Reproduce

```

```


## Expected Behavior



## Environment Information
<details open><summary><code>`conda info`</code></summary><p>


```

```
</p></details>


<details open><summary><code>`conda config --show-sources`</code></summary><p>


```

```
</p></details>


<details><summary><code>`conda list --show-channel-urls`</code></summary><p>


```

```
</p></details>

**Comment by dwervin** [comment]
If I would have expected that I would run into such an issue, I could have collected the details before and after.  Unfortunately, it's too late for that now.

The best I can tell you is that:

1. Downloaded and installed Anaconda a couple of months ago.
2. Ran into an issue in Jupyter Notebooks where the Shutdown buttons were not working.
3. Searched for this issue and found an article that suggested to update jupyter with Conda.  They emphasized that the update be done with Conda since it is supposed to do a better job of handling dependencies.  Sounded good to me as I prefer conda over pip anyway, plus that's what was used to install additional packages I needed that were not included in the Anaconda distribution.
4. I ran ```conda update jupyter``` per the suggestion in the article.  That's it.  It appeared to update several other packages, but conda knows what it's supposed to do so I was not concerned.
5. When it was complete, I started Jupyter, opedned a notebook, and confirmed that the shutdown buttons now worked.
6. I opened a notebook that I was working with before the update, but when I ran the cell to import packages I needed, I received the import error shown above.

Unfortunately, I don't have the before and after details that would be helpful to you for trying to find where and why conda went wrong.

In general, you guys have done a great job with conda to make managing dependencies easier, but it seems that conda did not manage things well in this case.  It is surprising since matplotlib has become a fundamental package on which so many other packages rely.  If conda can foul up matplotlib when using in a straightforward manner as I did, are there other steps I should take to prevent problems like this in the future?

Thanks
